queries.sql

```diff
@@ -8,3 +8,28 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT *FROM animals WHERE neutered = 'True';
 SELECT *FROM animals WHERE name != 'Gabumon';
 SELECT *FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+
+ROLLBACK;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.), Please verify that change was made. Then after the roll back verify that the species columns went back to the state before the transaction.

[Hint] You can do that by simply adding this query before and after the rollback:
```sql
SELECT species from animals;
```

Reply:
I get it now, Thanks.